src/common/image.c

```diff
@@ -502,15 +503,16 @@ void _pop_undo(gpointer user_data, const dt_undo_type_t type, dt_undo_data_t dat
 
       *imgs = g_list_prepend(*imgs, GINT_TO_POINTER(undogeotag->imgid));
       list = g_list_next(list);
+      i++;
     }
-
+    if(i > 1) dt_control_log((action == DT_ACTION_UNDO)
+                              ? _("undone geo location onto %d images")
```

Comment:
I would reword this as : "geo location undone for %d images"

As undone is a past participle I would put after the subject.

But we may want a review from a native English speaker.

(same below for redone)

Reply:
@elstoc : What do you think ? TIA.

Reply:
I've just adapted the control-job messages ("added time offset to ...", "set date time to...", "applied matched GPX location ...".
They may have to be revisited too ...

Reply:
@TurboGit I agree "geo location undone for %d images" is better. I've amended references to "date time" in the user manual to read "date/time" and I think they read better this way so, while you're changing it best to be consistent. The only reference I've left unchanged in dtdocs is the module control itself, so that'll need to be updated if you choose to change it here.

"redone" would probably read better as "re-applied to"

Reply:
done!

Reply:
@elstoc, I've changed all the date time to date/time here. 
I've a commit ready to align the date/time wording in dtdocs.

Reply:
Thanks @phweyland

Reply:
Another minor thing while we're discussing wording - I think "geo location" might read better if it's hyphenated "geo-location". No need to necessarily do that in this PR but something for consideration later (since it will possibly have implications elsewhere). I suppose more generally it's arguable whether the "geo" is required at all.